### PR TITLE
Hotspot show page

### PR DESCRIPTION
**Pull request by vicmgs**

### CODE REVIEW

File: assets/js/Router.jsx [hunk 1]
```diff
@@ -336,7 +336,7 @@ const Router = (props) => {
                                 <Profile user={user} {...props} />
                               )}
                             />
-                            {false && (
+                            {true && (
```
Comment (ambuznego) [inline assets/js/Router.jsx:339]: the game continues 😈

File: assets/js/components/coverage/CoverageHotspotShow.jsx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+                    <Tooltip title="Click to Edit Alias">
+                      {hotspotAlias || "No Alias"}
+                    </Tooltip>
+                  </span>
+                </Link>
+              )
+            }
+          </div>
+          <div style={{ marginBottom: 8 }}>
+            <Text style={{ fontSize: 15 }}>
```
Comment (ambuznego) [inline assets/js/components/coverage/CoverageHotspotShow.jsx:107]: bump to 16 to match design?

File: assets/js/components/coverage/CoverageHotspotShow.jsx [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+              This Hotspot has heard from of your devices over the past 30 days.
```
Comment (ambuznego) [inline assets/js/components/coverage/CoverageHotspotShow.jsx:108]: there should be a number of devices here, right? and an alternative msg (`none of`) for when there's no org packets heard by this hotspot?

Comment (ambuznego) [reply assets/js/components/coverage/CoverageHotspotShow.jsx:108]: also are we changing this to 2 days since i think that's how we're calculating stats?

Comment (vicmgs) [reply assets/js/components/coverage/CoverageHotspotShow.jsx:108]: will add the device stats next with the graph stuff in a separate PR.

File: assets/js/components/coverage/CoverageHotspotShow.jsx (hunk 2 above)
Comment (vicmgs) [reply assets/js/components/coverage/CoverageHotspotShow.jsx:107]: yes

File: assets/js/Router.jsx (hunk 1 above)
Comment (vicmgs) [reply assets/js/Router.jsx:339]: lul

**Review by ambuznego — APPROVED** [review]

**Pull request merged**